web3.js/src/connection.ts

```diff
@@ -3359,6 +3364,25 @@ export class Connection {
     return res.result;
   }
 
+  /**
+   * Check whether a blockhash is still valid or not
+   */
+  async isBlockhashValid(
+    blockhash: string,
+    commitment?: Commitment,
+    minContextSlot?: number
+  ): Promise<boolean> {
```

Comment:
This ‘multiple optional arguments’ signature can be troublesome if you want to supply a `minContextSlot` but you _don't_ want to supply a commitment.

```
const isValid = await isBlockhashValid('ABC', undefined, 1143);
```

What's `undefined`? Should I label it with a comment? Awkward!

Instead, use the `configOrCommitment` pattern (see `getProgramAccounts`).

Reply:
Refactored optional arguments using configOrCommitment